ibis/expr/types/generic.py

```diff
@@ -401,6 +401,27 @@ def fillna(self, fill_value: Scalar) -> Value:
         │ female     │
         └────────────┘
 
+        Returns
+        -------
+        Value
+            `self` filled with `fill_value` where it is `NULL`
+        """
+        return ops.Coalesce((self, fill_value)).to_expr()
+
+    @deprecated(as_of="10.0", instead="use fillnull instead")
+    def fillna(self, fill_value: Scalar) -> Value:
+        """Replace any null values with the indicated fill value.
+
+        Parameters
+        ----------
+        fill_value
+            Value with which to replace `NA` values in `self`
+
+        See Also
+        --------
+        [`Value.coalesce()`](./expression-generic.qmd#ibis.expr.types.generic.Value.coalesce)
+        [`ibis.coalesce()`](./expression-generic.qmd#ibis.coalesce)
+
```

Comment:
Should also update the implementation below to just call `self.fillnull` instead.